presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -845,15 +845,17 @@ void VeloxQueryPlanConverterBase::toAggregations(
           auto pos = functionId.find(";", start + 1);
           if (pos == std::string::npos) {
             auto argumentType = functionId.substr(start + 1);
-            aggregate.rawInputTypes.push_back(
-                stringToType(argumentType, typeParser_));
+            if (!argumentType.empty()) {
```

Comment:
Can you explain the bug? Can you share an example of how this manifests. Also, add a test.  If this is a bug fix, also consider pulling this out into a separate PR to merge earlier.

Reply:
Opened a PR: https://github.com/prestodb/presto/pull/24059